internal/ingress/controller/template/template.go

```diff
@@ -259,7 +259,7 @@ func buildResolvers(res interface{}, disableIpv6 interface{}) string {
 }
 
 // buildLocation produces the location string, if the ingress has redirects
-// (specified through the nginx.ingress.kubernetes.io/rewrite-to annotation)
+// (specified through the nginx.ingress.kubernetes.io/rewrite-target annotation)
```

Comment:
I would remove the line starting with "(specified..." completely. We don't need to be that specific in my opinion, these kind of comments typically get outdated and never changed.